src/Tizen.Content.MediaContent/Tizen.Content.MediaContent/MediaInfoCommand.cs

```diff
@@ -496,7 +504,26 @@ public bool Delete(string mediaId)
                 return false;
             }
 
-            CommandHelper.Delete(Interop.MediaInfo.Delete, mediaId);
+            Interop.MediaInfo.GetMediaFromDB(mediaId, out var handle).
+                ThrowIfError("Failed to delete MediaInfo.");
+
+            if (handle.IsInvalid)
+            {
+                return false;
```

Comment:
According to your command, it should throw exception

Reply:
Yes, you're right. I've checked it with native developer and there's no case for null handle without exception.
I'll remove it.